ci-operator/config/openshift/installer/openshift-installer-master__altinfra.yaml

```diff
@@ -119,6 +119,20 @@ tests:
     cluster_profile: aws
     workflow: openshift-e2e-aws-proxy
   timeout: 6h0m0s
+- always_run: false
+  as: e2e-aws-ovn-public-ipv4-pool
```

Comment:
nit: you might want to throw `capi` into the name. although not a requirement--altinfra should be sufficient

Reply:
hmm .. yeah, good call.  i'll make that change